src/op/tensorize.cc

```diff
@@ -222,8 +231,9 @@ class TensorIntrinMatcher final : public IRMutator {
       CHECK_GE(e.region.size(), e.tensor.ndim());
       // Enable fuzzy matching, to match [1, n, m] to [n, m]
       e.start = e.region.size() - e.tensor.ndim();
-      for (size_t i = 0; i < e.start; ++i) {
-        CHECK(is_one(e.region[i]->extent))
+      for (size_t j = 0; j < e.start; ++j) {
+        auto canonical_extent = Simplify(e.region[j]->extent, *compute_intrin_iter_space);
```

Comment:
if we have a analyzer, we don't have to call Simplify,  but instead we can call analyzer_->Simplify

Reply:
Actually here, I think it is going to be tricky. I suppose that is the reason dom_map constructed and passed from outside.

Reply:
As an alternative, we could create an Analyzer outside, populate the iteration space(via binding) using dom_map, and analyzer->Bind, then pass the analyzer pointer in.

Reply:
That sounds good for consolidating all the simplify stuff, however I think that would be a bit of hefty change and I was wondering if we can split it into a different PR.